Week5/solution/Week5_Solution.py

```diff
@@ -0,0 +1,33 @@
+# INSERTING RECORDS
+def insertFunct():
+    import csv    #importing csv functions
+    fields=['sid','sname','city','contact']
+    rows=[[101,'Om','Bardoli',9999990000],
+      [102,'Sai','Surat',9998888000],
+      [103,'Ram','Navsari',9991110000],
+      [104,'Vatsal','Vyara',9999666000],
+      [105,'Harsh','Bardoli',8889900222]]     #pre-inserted records
+    print("Enter the required records: ")
+    for i in range(5):
+        s_id=int(input("Enter Student ID: "))
+        name=input("Enter Student Name: ")
+        city=input("Enter their city name: ")
+        contact=int(input("Enter their Contact info: "))
+        srow=[s_id,name,city,contact]         #user input records
+        rows.append(srow)
```

Comment:
Without append( ) method , how can we insert user input record ?

Reply:
we can use extend() however thats more preferable for more than one item

Reply:
we can use '+' sign to do it as well
```
L=[]
tup=(member_list)
L=L+tup
```